packages/composites/apy-finance/README.md

```diff
@@ -6,16 +6,39 @@ The adapter calculates APY Finance total value locked
 
 The adapter takes the following environment variables:
 
-- `RPC_URL`: Blockchain RPC endpoint to get the needed on-chain data
-- `REGISTRY_ADDRESS`: Address Registry contract address used to query the Chainlink Registry
-- `DATA_PROVIDER_URL`: The adapter URL for the price data provider
-- `DEFAULT_QUOTE` (Optional): Currency that the price will be fetched by default. `USD` used by default
+To be functional the location of a running Token Allocation composite adapter must be configured.
+
+| Required? |                 Name                 |                       Description                       | Options | Defaults to |
+| :-------: | :----------------------------------: | :-----------------------------------------------------: | :-----: | :---------: |
+|    ✅     | `TOKEN_ALLOCATION_DATA_PROVIDER_URL` |  The location of a Token Allocation composite adapter   |         |             |
```

Comment:
This composites should rely on Token Allocation Code, not Data. There's no need to call it, just use it.